web-console/src/lib/components/streaming/management/PipelineTable.tsx

```diff
@@ -468,7 +473,7 @@ export default function PipelineTable() {
     {
       field: 'modification',
       headerName: 'Changes',
-      flex: 1,
+      width: 145,
```

Comment:
I don't really understand how this layout stuff works across all screen sizes, but what is 145?

Reply:
There are two options in the context of that grid:
1) Have column width scale with a certain percentage of total available width, and
2) Have a fixed column width in pixels no matter the row width.
That specific column displays a near-fixed width info, so when playing with the UI I decided to have the width fixed by default, otherwise when making the window narrower part of the cell was getting hidden.
I eyeballed the number of logical (not physical) pixels - 145 - and logical pixels scale with e.g. display pixel density, so this value looks decent across different display sizes